app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/DropDownMentionsSuggestions.kt

```diff
@@ -143,10 +146,12 @@ private const val MINIMUM_SCREEN_HEIGHT = 700
 private val DROP_DOWN_HEIGHT_SMALL = 150.dp
 private val DROP_DOWN_HEIGHT_LARGE = 200.dp
 private const val FIRST_LINE_INDEX = 0
-private const val HALF_SCREEN = 2.5
-private const val EIGHTY_PERCENT = 0.80
+private const val HALF_SCREEN = 0.45
+private const val SIXTY_PERCENT = 0.75
+private const val SEVENTY_PERCENT = 0.75
```

Comment:
These two should be the same? Maybe we can reduce it and use only one of them?

Reply:
Oh! that's wrong. should not the same